tests/fv/__init__.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sh
+from sh import docker
+
+
+def setup_package():
+    pwd = sh.pwd().stdout.rstrip()
+
+    docker_exec = docker.bake("exec")
+    host1_exec = docker_exec.bake("-t", "host1", "bash", "-c")
```

Comment:
As it stands, docker-jenkins is *only* allowed to run the docker-fvs job. So it seems quite reasonable to interpolate the entire list of containers into this.